### PR TITLE
fixing funky avatars in /team

### PR DESCRIPTION
**Pull request by tobyab**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/team**](https://site-gzeby04vm.hackclub.dev/team) |
|:-:|
|<a href="https://site-gzeby04vm.hackclub.dev/team">Screenshot of /team</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/j6MpPbE-zVvRvbKXTcuC6Bi4MebAjdYcMVPcCXwt4vT4ZDlFVT4AwgIVJk8L_0iNq_yETxkwkkaEwRhhrhGtfw.png">(view full size)</a> |







Commit ed40f39b3681cc2a075554f52a1452647eeda44b (https://site-gzeby04vm.hackclub.dev).

File: components/bio.js [hunk 1]
```diff
@@ -22,10 +22,10 @@ export default function Bio({ popup = true, spanTwo = false, ...props }) {
           maxWidth: '600px',
           zIndex: !popup ? 1003 : 5,
           maxHeight: '90vh',
-          overflowY: 'scroll',
+          overflowY: 'hidden',
```
Comment (grymmy) [inline components/bio.js:25]: Just curious, what necessitated this change?

**Review by grymmy — APPROVED** [review]
Lgtm

File: components/bio.js (hunk 1 above)
Comment (tobyab) [reply components/bio.js:25]: Whilst connected to an external monitor & in clamshell mode, (I'm guessing there are other cases, too), scrollbars are shown on browsers running on MacOS (not sure about Windows). The Card component has an inner scrollbar, so this hides that!
Here's a before & after!
SCR-20230926-peiy
image

**Pull request merged**